.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: "composer"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    allow:
+      - dependency-type: "all"
+    commit-message:
+      prefix: "[Laravel]"
```

Comment:
Waarom Laravel?

Reply:
Ik had deze comment over het hoofd gezien en dit was een verkeerde prefix. Ik was teveel bezig met Laravel in die weken en heb uit automatisme Laravel als prefix gezet. Ik pas 'm aan en tag je even!`